source/platforms/databricks.md

```diff
@@ -1,12 +1,42 @@
 # Databricks
 
-## Databricks Notebooks
-
 You can install RAPIDS libraries into a Databricks GPU Notebook environment.
 
-### Launch a single-node Databricks cluster
+## DASK Rapids in Databricks MNMG Cluster
+
+You can launch Dask RAPIDS cluster on a multi-node GPU Databricks cluster. To do this, you must first create an [initialization script](https://docs.databricks.com/en/init-scripts/index.html) to install Dask before launching the Databricks cluster.
+
+Databricks recommends storing all cluster-scoped init scripts using workspace files. Each user has a Home directory configured under the `/Users` directory in the workspace. Navigate to your home directory in the UI and select **Create** > **File** from the menu, create an `init.sh` script with contents:
+
+```bash
+#!/bin/bash
+set -e
+
+# The Databricks Python directory isn't on the path in
+# databricksruntime/gpu-tensorflow:cuda11.8 for some reason
+export PATH="/databricks/python/bin:$PATH"
+
+# Install RAPIDS (cudf & dask-cudf) and dask-databricks
+/databricks/python/bin/pip install --extra-index-url=https://pypi.nvidia.com \
+      bokeh==3.2.2 \
+      cudf-cu11 \
+      dask[complete] \
+      dask-cudf-cu11 \
+      dask-cuda==23.10.0 \
```

Comment:
We probably don't want to hard-code this here, but without the pin we sometimes get `0.18.0` which is a bit weird (@pentschev FYI). We should use the `{rapids_version}` template variable here instead of a hard value.

Reply:
My best guess is this is happening because in the early ages we had no upper limit on Dask/Distributed versions in Dask-CUDA, so this probably happens because some other package is pulling a newer Dask version than what the latest `23.10.0` is pinned to. I don't know if/how we could fix this though. @vyasr has a vast knowledge of PyPI wheels so maybe he will have good ideas here.

Reply:
Could we yank the old releases without the upper limit from PyPI? It's hard to imagine anyone will be using them this many years down the line.

Reply:
It has some historical value, for example in https://developer.nvidia.com/blog/debugging-mixed-python-and-c-language-stack/ we did exactly that, recreated the environment with the proper (old) versions to serve as debugging basis. I agree people should not be using them in a daily basis, but debugging and benchmarking are still valid use cases in my book.

If yanking the packages is the only way forward, I think this is something that should be raised internally for more discussion before we do it to make sure we're not breaking anything else.

Reply:
I wish we could patch PyPI metadata like conda-forge...

Reply:
If I'm understanding the problem correctly, instead of using an `==` could you just use a lower bound `dask-cuda >= {rapids_version}`? Or maybe using `=={rapids_version}` is sufficient? I may not be fully understanding the problem we're trying to solve.